### PR TITLE
test: updating example configs for upcoming disallowed by default features

### PR DESCRIPTION
**Pull request by alyssawilk**
From the lamentably dead flip PR, replacement in progress

Risk Level: n/a
Testing: yes
Docs Changes: n/a

### CODE REVIEW

**Comment by snowp** [comment]
Seems like CI is failing

**Comment by alyssawilk** [comment]
Yeah, someone else checked in bad config since then.  Bah!  I knew I should have gotten this in before the break!

**Comment by dio** [comment]
Seems like the `configs/envoy_service_to_service_v2.template.yaml` still has at least three non-`typed_config`?

File: examples/jaeger-tracing/front-envoy-jaeger.yaml [hunk 1]
```diff
@@ -67,6 +66,7 @@ tracing:
       collector_cluster: jaeger
       collector_endpoint: "/api/v1/spans"
```
Comment (dio) [inline examples/jaeger-tracing/front-envoy-jaeger.yaml]: nit. `/api/v2/spans`

File: examples/jaeger-tracing/service1-envoy-jaeger.yaml [hunk 2]
```diff
@@ -107,6 +105,7 @@ tracing:
       collector_cluster: jaeger
       collector_endpoint: "/api/v1/spans"
```
Comment (dio) [inline examples/jaeger-tracing/service1-envoy-jaeger.yaml]: nit. `/api/v2/spans`

File: examples/jaeger-tracing/service2-envoy-jaeger.yaml [hunk 3]
```diff
@@ -64,6 +63,7 @@ tracing:
       collector_cluster: jaeger
       collector_endpoint: "/api/v1/spans"
```
Comment (dio) [inline examples/jaeger-tracing/service2-envoy-jaeger.yaml]: nit. `/api/v2/spans`

File: examples/zipkin-tracing/front-envoy-zipkin.yaml [hunk 4]
```diff
@@ -72,6 +71,7 @@ tracing:
       "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
       collector_cluster: zipkin
       collector_endpoint: "/api/v1/spans"
```
Comment (dio) [inline examples/zipkin-tracing/front-envoy-zipkin.yaml]: nit. `/api/v2/spans`

File: examples/zipkin-tracing/service2-envoy-zipkin.yaml [hunk 5]
```diff
@@ -63,6 +62,7 @@ tracing:
       "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
       collector_cluster: zipkin
       collector_endpoint: "/api/v1/spans"
```
Comment (dio) [inline examples/zipkin-tracing/service2-envoy-zipkin.yaml]: nit. `/api/v2/spans`

File: examples/zipkin-tracing/service1-envoy-zipkin.yaml [hunk 6]
```diff
@@ -106,6 +104,7 @@ tracing:
       "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
       collector_cluster: zipkin
       collector_endpoint: "/api/v1/spans"
```
Comment (dio) [inline examples/zipkin-tracing/service1-envoy-zipkin.yaml]: nit. `/api/v2/spans`

**Pull request merged**